src/main/java/org/springframework/cloud/scheduler/spi/kubernetes/KubernetesSchedulerProperties.java

```diff
@@ -77,6 +77,27 @@
 	 */
 	private String namespace = KUBERNETES_NAMESPACE;
 
+	/**
+	 * The Secret name to use when pulling a private docker image.
```

Comment:
small `S` on Secret.  Nitpicky I know :-)

Reply:
references a `Secret` object, so keeping as-is per discussion